### ISSUE TITLE
[BUG] MKS TFT32 L V3.0 Screen H Flip

### ISSUE THREAD

**Issue by stef-ladefense** [bigtreetech/BIGTREETECH-TouchScreenFirmware#2720]
hi,
I just did a test on an MKS 32L V3.0 that I had in a drawer.

I simply took the following basic files:
config.ini
MKS_TFT28_V3.0.27.x.bin renames MKSTFT28.bin
folder TFT28

the MKS TFT32 shares the firmware of the MKS TFT28 and must therefore be named the same for the firmware update to work.

the update works until the end but the displays are mirrored horizontally and vertically at 180°.
at the end the touch adjustment procedure does not follow the inversions of the screen, it is therefore necessary to touch in the opposite diagonal.

![20230217_094447](https://user-images.githubusercontent.com/43587190/219601575-df38b8ae-b2c1-42bd-a545-9038324e6e19.jpg)

I put the additional photos in the next post

**Comment by stef-ladefense** [comment]
![20230217_094130](https://user-images.githubusercontent.com/43587190/219601922-5973295f-0e9d-4ae8-8e22-0a7aa2a6ac53.jpg)
![20230217_094140](https://user-images.githubusercontent.com/43587190/219601996-dc393c7b-401b-4cca-9977-8afb94724e43.jpg)
![20230217_094205](https://user-images.githubusercontent.com/43587190/219602034-899de243-f040-49ab-8467-69f0cab30adb.jpg)
![20230217_094228](https://user-images.githubusercontent.com/43587190/219602134-b170a544-80c0-4cc2-95b5-f6ec1a147124.jpg)
![20230217_094253](https://user-images.githubusercontent.com/43587190/219602152-a445f73f-0d4e-44b1-8e3b-2623203ed29f.jpg)
![20230217_094332](https://user-images.githubusercontent.com/43587190/219602166-c8efd399-8164-404b-ab10-c370e6643a31.jpg)
![20230217_094426](https://user-images.githubusercontent.com/43587190/219602177-24c675ee-d378-4992-a21b-fad63e94c4a6.jpg)

I hope it will help to fix the firmware

Stéphane

**Comment by radek8** [comment]
Did you download the update files from the Marter branch?

**Comment by radek8** [comment]
https://github.com/bigtreetech/BIGTREETECH-TouchScreenFirmware/issues/2436

**Comment by stef-ladefense** [comment]
@radek8 
it's MKS display and the .bin has 2 day, 2.15.2023 10;53 from master

**Comment by radek8** [comment]
Also try resetting the display with an empty reset.txt file

**Comment by stef-ladefense** [comment]
I applied without effect

**Comment by radek8** [comment]
The problem will be that your display is not among the supported versions of MKS displays

![image](https://user-images.githubusercontent.com/46979052/219877374-758d9ebd-5998-4c31-90c4-c5db5507c307.png)

**Comment by stef-ladefense** [comment]
yes but the MKS TFT32 V3.0 has the same hardware as the MKS TFT28 V3.0 so it shouldn't be very complicated to correct.
I will look at the sources and make the correction as soon as I have some time.

**Comment by radek8** [comment]
MKS TFT 35 V1 uses
#define TFTLCD_DRIVER (ILI9488 | NT35310)

MKS TFT 28 V3 is used
#define TFTLCD_DRIVER ILI9325

**Comment by radek8** [comment]
You can try it
MKS_TFT32_V1_3
